usecases/objects/batch_references_add.go

```diff
@@ -117,6 +127,110 @@ func (b *BatchManager) validateReference(wg *sync.WaitGroup, ref *models.BatchRe
 	}
 }
 
+func validateReferenceMultiTenancy(ctx context.Context,
+	principal *models.Principal, schemaManager schemaManager,
+	repo VectorRepo, source *crossref.RefSource, target *crossref.Ref,
+	tenantKey string,
+) error {
+	if source == nil || target == nil {
```

Comment:
Does this mean that `source != nil || target == nil` and `source == nil || target != nil`?